Mathlib/MeasureTheory/Function/LpSeminorm/Basic.lean

```diff
@@ -143,6 +143,9 @@ theorem eLpNorm'_measure_zero_of_exponent_zero {f : α → ε} : eLpNorm' f 0 (0
 theorem eLpNorm'_measure_zero_of_neg {f : α → ε} (hq_neg : q < 0) :
     eLpNorm' f q (0 : Measure α) = ∞ := by simp [eLpNorm', hq_neg]
 
+omit [ENormedAddMonoid ε]
```

Comment:
Can you put this variable in a `section`, we're trying to move away from the `omit` style (unless it's `omit ... in`).

Reply:
Sure, done!